code/game/machinery/vending/marine_vending.dm

```diff
@@ -552,6 +553,7 @@
 						/obj/item/attachable/stock/rifle = 3 ,
 						/obj/item/attachable/stock/revolver = 3,
 						/obj/item/attachable/stock/smg = 3,
+						/obj/item/attachable/stock/tactical = 4,
```

Comment:
Same here. Make unusual stocks similar or inferior in numbers in comparison with the usual ones.